content/en/docs/File Format/implementationstatus.md

```diff
@@ -118,7 +118,7 @@ Implementations:
 
 | Feature                                   | arrow | parquet-java  | arrow-go | arrow-rs | cudf  | hyparquet | duckdb |
 | ----------------------------------------- | ----- | ------------- | -------- | -------- | ----- | --------- | ------ |
-| External column data (1)                  |  ✅   |  ✅           |  ❌      |  ❌      |  (W)  |  ❌       |   ❌   |
+| External column data (1)                  |  ✅   |  ✅ (*)       |  ❌      |  ❌      |  (W)  |  ✅       |   ❌   |
```

Comment:
Why adding * to parquet-java?

Reply:
Sorry that wasn't meant for this PR! I'll revert that as it is a separate from my intended update here.

Backstory: I've been implementing support for parquet `file_path` for external column data in hyparquet. And the parquet-java implementation has significant limitations: 1) if any file_path is set, then _every_ column chunk must be external, 2) all file_paths that are external must be the same file, and 3) even with those assumptions I've been unable to make a single example parquet file with external column data that it _can_ read. That might be on me, I'm still investigating.

Reply:
Citation for parquet-java partial `file_path` support: [ParquetMetadataConverter.java#L1814](https://github.com/apache/parquet-java/blob/f3535a6a6/parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java#L1814)

Reply:
What about clarifying that in a separate PR?

Reply:
Reverted the change to parquet-java in this PR. I will open a new PR with the java update later. Thanks for catching that!